doc/architecture/decisions/003-authoring-schema-variants.md

```diff
@@ -0,0 +1,41 @@
+# 3. Generate Schema variants from a data source + manifest
+
+Date: 04/07/2017
+
+## Status
+
+Proposed
+
+## Context
+
+Schema variations are tackled through a mixture of routing rules (go here if...), skip conditions (skip this if...) and splitting of schemas into "form types" (e.g. RSI form types 0102, 0112 are survey runner schemas 1_0102.json and 1_0112.json). Compared to existing surveys MBS has a large number of form types which makes splitting and maintaining them by hand too onerous. The Author service is the long term solution but we need something more manageable in the interim to make management of MBS content more cost effective.
+
+We need a solution that lightens the load of authoring form variations.
+
+## Decision
+
```

Comment:
I think it might be clearer to have one high level decision - we will split schemas out into manifests and blocks and generate the survey runner JSON at build time - and make each decision point below a list item.